packages/azure-services/src/system-config.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { injectable } from 'inversify';
+
+@injectable()
+export class SystemConfig {
+    public readonly storageName: string = process.env.AZURE_STORAGE_NAME;
```

Comment:
it might be better to not use AZURE_STORAGE_NAME as this is set only after QueueServiceURLProvider or BlobServiceClientProvider is used. Also, caching secrets in general can be solved by adding a caching layer for all the fetched secrets.

Reply:
used in conjunction with queue storage hence it will be initialized